platform/ext/target/nrf5340/target_cfg.h

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2017-2019 Arm Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __TARGET_CFG_H__
+#define __TARGET_CFG_H__
+
+#include "platform/ext/common/uart_stdout.h"
+#include "tfm_plat_defs.h"
+
+
+/**
+ * \brief Store the addresses of memory regions
+ */
+struct memory_region_limits {
+    uint32_t non_secure_code_start;
+    uint32_t non_secure_partition_base;
+    uint32_t non_secure_partition_limit;
+    uint32_t veneer_base;
+    uint32_t veneer_limit;
+#ifdef BL2
+    uint32_t secondary_partition_base;
+    uint32_t secondary_partition_limit;
+#endif /* BL2 */
+};
+
+/**
+ * \brief Configures the Security Protection Unit.
+ *
+ * \return  Returns error code.
+ */
+int32_t spu_init_cfg(void);
```

Comment:
I can't find that one declared/defined for other targets nor any references to it. Same story with `spu_clear_irq`. Are these indeed needed?

Reply:
I think so. they are defined, for example, for AN521 platform, for the MPC, PPC controllers.
I guess it is a way to enable the interrupt and initialize the security unit. Might well be empty functions for us. I kept them for now.

Reply:
Okay, I see my mistake. I was looking for that exact name, but this function should reflect the actual name of the controller, which is other for our target than for all the others. Sorry for the noise.

Reply:
that's fine - thanks for looking around the details